en/admin-guide/datamodel-tool/index.rst

```diff
@@ -0,0 +1,74 @@
+Data-model tool
+===============
+
+The plugin contains a data-model tool capable of initializing and upgrading a QGEP database.
+
+It is aimed at system administrators and power users, and should not be used by end users, as it
+could lead to data loss if not used properly.
+
+For advanced cases, it is also possible to `upgrade your database manually <../db-update-pum/index.html>`_
+
+Setup
+-----
+
+The data-model tool is hidden by default. To make it visible, you must first enable ``Admin mode`` 
+in the plugin settings :
+
+* Go to ``Plugin > QGEP > Settings``
+
+* Switch to the ``Developer options``
+
+* Check ``Admin mode``
+
+* Restart QGIS
+
+The data-model tool should now appear under ``Plugin > QGEP > Datamodel``
+
+Usage
+-----
+
+Main dialog
+^^^^^^^^^^^
+
+The main dialog looks like this.
+
+.. figure:: images/datamodel-tool.png
+
+* **Release**:
+
+    * **Version**: the version to retrieve the files from. Unless ``developper mode`` is enabled in the QGEP settings, only the latest stable version is available. Selecting anything different from the latest stable version should only ever be used on a testing database, as it could upgrade the database to an inconsistent state.
```

Comment:
```suggestion
    * **Version**: the version to retrieve the files from. Unless ``developer mode`` is enabled in the QGEP settings, only the latest stable version is available. Selecting anything different from the latest stable version should only ever be used on a testing database, as it could upgrade the database to an inconsistent state.
```